### PR TITLE
Improved expression of anchors

### PR DESCRIPTION
**Pull request by gmlwhdtjd**
The structure of Anchors has been reorganized to improve ambiguous expressions and expressions that were previously impossible.

Ambiguous expressions such as the following are no longer available.
```swift
// Ambiguous Attribute Chaining
// This means ".leading == superview.leading + 50, .bottom == superview.bottom + 50"
Anchors
    .leading.equalToSuper()
    .bottom.equalToSuper(constant: 50)

// Ambiguous omission
// This means "Anchors.width.equalToSuper().constant(50)" statement 
// not "Anchors.width.equalTo(constant: 50)"
Anchors.width.constant(50) 
```

In addition, it provides an improved syntax for chaining different types of attributes such as:
```swift
Anchors.centerX.centerY.width.height.equalToSuper()

Anchors.leading.top.bottom.equalTo(someView, constant: 50)
```

Finally, an inwardOffset version has been added to all methods that can input constants.
```swift
// This means 
// .leading == superview.leading + 50
// .top == superview.top + 50
// .bottom == superview.bottom + 50
Anchors.leading.top.bottom.equalToSuper(constant: 50)

// This means 
// .leading == superview.leading + 50
// .top == superview.top + 50
// .bottom == superview.bottom - 50
Anchors.leading.top.bottom.equalToSuper(inwardOffset: 50)
```

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.4
+// swift-tools-version:5.7
```
Comment (gmlwhdtjd) [inline Package.swift:1]: From now on, SwiftLayout requires swift 5.7.

We need to manage the swift version.

**Review by della-padula — APPROVED** [review]
Thank you for your effort. 👍

File: Package.swift (hunk 1 above)
Comment (gmlwhdtjd) [reply Package.swift:1]: @oozoofrog @della-padula 
I will put a table in README.md that summarizes the versions of Swift that can be used for each version.
In the case of the latest version, only 5.7 will be supported.

**Comment by oozoofrog** [comment]
👍 wonderful

**Pull request merged**